### PR TITLE
*: Update links to beyondstorage at matrix

### PR DESCRIPTION
**Pull request by Prnyself**
Signed-off-by: Prnyself <281239768@qq.com>

### CODE REVIEW

**Comment by Prnyself** [comment]
ref: https://github.com/beyondstorage/community/issues/29

File: community/index.md [hunk 1]
```diff
@@ -25,10 +25,10 @@ Our formal announcement will be on our forums.
 
 This is the main method to communicate with Beyond Storage developers.
 
-We host our matrix server on `https://chat.aos.dev`.
+We host our matrix server on `https://chat.matrix.org`.
```
Comment (Xuanwo) [inline community/index.md:28]:
```suggestion
We host our matrix server on `https://matrix.org`.
```

File: i18n/zh-CN/docusaurus-plugin-content-docs-community/current/index.md [hunk 2]
```diff
@@ -25,10 +25,10 @@ Our formal announcement will be on our forums.
 
 This is the main method to communicate with Beyond Storage developers.
 
-We host our matrix server on `https://chat.aos.dev`.
+We host our matrix server on `https://chat.matrix.org`.
```
Comment (Xuanwo) [inline i18n/zh-CN/docusaurus-plugin-content-docs-community/current/index.md:28]: Please don't update files under i18n.

Comment (Prnyself) [reply i18n/zh-CN/docusaurus-plugin-content-docs-community/current/index.md:28]: My fault, I just search it globally and modify all of them.

**Review by Xuanwo — APPROVED** [review]

**Pull request merged**